src/channel.cpp

```diff
@@ -611,8 +614,8 @@ EPutDataStat CChannel::PutAudioData ( const CVector<uint8_t>& vecbyData,
                     eRet = PS_AUDIO_ERR;
                 }
 
-                // manage audio fade-in counter
-                if ( iFadeInCnt < iFadeInCntMax )
+                // manage audio fade-in counter, after channel is identified
+                if ( iFadeInCnt < iFadeInCntMax && bIsIdentified )
```

Comment:
Why is this _not_ added to the `if` at line 600?

Reply:
Because that would all require much more understanding and testing for any side-effects. This was just a simple way of keeping the channel gain at zero until the channel is identified.

We still need to be able to process a new connection a few lines further down.

Reply:
OK